firmware-management/updated/updated-client/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
I think we should add the copyright notice and SPDX License identifier to this file.

Reply:
4c1a07c